src/components/Plan.js

```diff
@@ -68,20 +83,31 @@ class Plan extends Component {
                         phases.map(phase => {
                             return (
                                 <li className="phase" key={ phase }>{ phase }
-                                    <ul>
-                                        {
-                                            phasedIssues[phase].map(issue =>
-                                                <li className="task" key={ issue.number }>
-                                                    <a href={ issue.url } target="_blank" rel="noopener noreferrer" >{ `${issue.number} ${issue.title}` }</a>
-                                                    <span className={ 'state ' + issue.state }>
-                                                    {
-                                                        issue.state === 'done' ? ' (done)' : issue.state === 'wip' ? ' (in progress)' : ''
-                                                    }
-                                                    </span>
+                                <ul>
+                                    {
+                                        repos.map(repo => {
+                                            if (phasedIssues[phase][repo].length === 0) return null;
+                                            return (
+                                                <li className="repo" key={ repo }>{ repo }
+                                                    <ul>
+                                                        {
+                                                            phasedIssues[phase][repo].map(issue =>
+                                                                <li className="task" key={ issue.number }>
+                                                                    <a href={ issue.url } target="_blank" rel="noopener noreferrer" >{ `${issue.number} ${issue.title}` }</a>
+                                                                    <span className={ 'state ' + issue.state }>
+                                                                    {
+                                                                        issue.state === 'done' ? ' (done)' : issue.state === 'wip' ? ' (in progress)' : ''
```

Comment:
We have reached peak code indentation in this file I think... 😁 It would probably be easier to read and review if this whole block moves to a function such `renderPhasedIssues` or something, and then you can call that here with `{ this.renderPhasedIssues(phase, repo) }` or similar.

Reply:
I extracted the layering out to an arbitrary depth issue tree layering component so this should be addressed.